packages/front-end/pages/admin.tsx

```diff
@@ -176,6 +207,35 @@ function OrganizationRow({
             <FaPencilAlt />
           </a>
         </td>
+        <td className="p-0 text-center">
+          <Tooltip
+            body={
+              hasGrowthbookClickhouse
+                ? "Already has an Inbuilt Growthbook Datasource"
+                : "Create Inbuilt Growthbook Clickhouse DataSource"
+            }
+          >
```

Comment:
Not for this PR, but have we considered adding these ternaries under an `enabled/disabled` prop to `Tooltip`? Seems like that could be useful in other places too

Reply:
Not sure if that is really needed.  In this case we want different messages if what it is wrapping is disabled or not.  Would we just ignore the `body` property in your suggestion?  The ternary seems simple enough.

Reply:
Maybe it's overkill, but i was thinking of it having two props for each of the items with a ternary. So an enabledBody, disabledBody etc. On second thought it probably clutters things more than it helps